astropy/units/astrophys.py

```diff
@@ -124,6 +124,13 @@
     doc="Rydberg: Energy of a photon whose wavenumber is the Rydberg constant",
     format={"latex": r"R_{\infty}", "unicode": "R∞"},
 )
+def_unit(
+    ["foe", "Bethe", "bethe"],
```

Comment:
Is capitalized `bethe` really necessary? I don't see that in https://en.wikipedia.org/wiki/Foe_(unit)

Reply:
I seem to remember seeing both in the literature; but unfortunately, since I don’t have a concrete paper in mind and most supernova neutrino papers cite papers by Bethe, that’s very hard to search for. I can ask a few colleagues what their gut feeling is.

But also, it looks to me like eponymous units typically include both spellings in AstroPy. Just the first few examples I’ve tested:
```Python
import astropy.units as u
u.Newton == u.newton  # True
u.Watt == u.watt  # True
u.Rayleigh == u.rayleigh  # True
u.Siemens == u.siemens  # True
```
So I’d err on the side of including both `bethe` and `Bethe`, too.